src/MIDebugEngine/ResourceStrings.Designer.cs

```diff
@@ -118,6 +118,17 @@ internal class ResourceStrings {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to Attaching to process {0} with {1} failed because of insufficient privilages with error message &apos;{2}&apos;.
+        ///
+        ///To attach to application on Linux, login as super user or disable ptrace..
```

Comment:
>disable ptrace [](start = 69, length = 14)

We aren't actually disabling ptrace but rather allowing ptrace to be used across user accounts. 

I wonder if we want an aka.ms link for directions?